base_tier_validation/models/tier_definition.py

```diff
@@ -134,7 +134,7 @@ def _get_review_needing_reminder(self):
         return self.env["tier.review"].search(
             [
                 ("definition_id", "=", self.id),
-                ("status", "=", "pending"),
+                ("status", "=", "waiting"),
```

Comment:
should not this be 
```
("status", "in", ("waiting", "pending",))
```
The waiting status was added and it is not replacing the pending status